generators/typescript-multiple-files/model.handlebars

```diff
@@ -1,3 +1,4 @@
+/* tslint:disable */
```

Comment:
Its just so the resulting typescript file doesn't get linted not the handlebars file. It would probably be better if there was an option to add this line through the CLI.